Makefile

```
@@ -205,6 +205,7 @@ helm-push:
 	    artifacthub-repo.yml:application/vnd.cncf.artifacthub.repository-metadata.layer.v1.yaml
 
 test:
+	$(GO_CMD) env -w GOTOOLCHAIN="$$($(GO_CMD) env GOVERSION )+auto" && \
```

Do we really need to write it permanently to go config?

```suggestion
	GOTOOLCHAIN="$$($(GO_CMD) env GOVERSION )+auto" \
```

I'd also suggest to add a NOTE/TODO here (with a ref to the upstream issue) to remove the hack when go has been fixed.

---

sounds good, adding it....

---

actually, it is possible indeed. Please disregard.